src/routes/Settings.tsx

```diff
@@ -28,6 +32,27 @@ export const SettingsRoute: React.FC = () => {
     ipcRenderer.invoke('get-app-version').then((result: string) => {
       setAppVersion(result);
     });
+
+    (async () => {
+      const response = await apiRequestAuth(
+        `${generateGitHubAPIUrl(Constants.DEFAULT_AUTH_OPTIONS.hostname)}`,
+        'GET',
+        accounts.token,
+      );
+
+      console.log(JSON.stringify(response.headers));
+      const missingScopes = Constants.AUTH_SCOPE.filter((scope) => {
+        return !response.headers['x-oauth-scopes'].includes(scope);
+      });
+      if (missingScopes.length > 0) {
+        new Notification('Gitify - Permissions', {
+          body:
+            'You need to grant all the permissions to use this app. Missing scopes: ' +
```

Comment:
Should this include an instruction of what they should do next?

Reply:
Ah I'd thought that you weren't logging them out? From "Opted by not logging out to not be too annoying."

Reply:
@bmulholland can you take another look? Now I'm not polluting the notifications, just disabling the option if the repo scope is missing. I'm also memoizing with the token as dependency so that we're not constantly checking for the scope.